doc/en/how-to/tmp_path.rst

```diff
@@ -8,9 +8,13 @@ How to use temporary directories and files in tests
 The ``tmp_path`` fixture
 ------------------------
 
-You can use the ``tmp_path`` fixture which will
-provide a temporary directory unique to the current test,
-created in the `base temporary directory`_.
+You can use the ``tmp_path`` fixture which will provide a temporary directory
+unique to each test function.
+
+By default, ``pytest`` retains the temporary directory for the last 3 ``pytest``
```

Comment:
This looks good, I would just move this to the end of this session (before line `69`), as this seems more like a detail best left for after we explain how to use the fixture.

Reply:
Agreed and updated.